src/EditorFeatures/TestUtilities/Workspaces/TestWorkspaceFixture.cs

```diff
@@ -128,19 +128,5 @@ private void CloseTextView()
             var existingProperties = (List<VisualStudio.Text.Formatting.TextFormattingRunProperties>)existingPropertiesField.GetValue(null);
             existingProperties.Clear();
         }
-
-        private static bool TryParseXElement(string input, out XElement output)
-        {
-            try
-            {
-                output = XElement.Parse(input);
```

Comment:
It almost seems like there should be two different APIs here, one where you pass in xml string content and throws if it can't be parsed.  Then another that passes just code content and doesn't try to parse xml.  But this API may be too entrenched to do that now.

Reply:
> The problem with this approach is if you have a typo in your XML

Please write tests for tests.

Reply:
Yes, this absolutely should have been two different APIs, but alas that ship has somewhat sailed. Definitely out of scope for this PR.

Reply:
Also:

> Please write tests for tests.

How do I test the tests for the tests??